scalableminds/webknossos
# Open assignments counter

### Mailable description of changes:
 - Improved task assignment performance ("get next task")

### Steps to test:
- create tasks with several instances
- get tasks as some user
- check, that remaining instances are decremented
- check, that no more tasks are assigned, than are available
- check, that `OpenAssignment` object is removed on subsequent task request

### Issues:
- fixes #1987

------
